### PR TITLE
add podSecurity docs for 1.8

### LINKED ISSUE #604: [Enhancement] Extend Pod Security Admission
### Description

Add documentation for the new rule `validate.PodSecurity` that [extends Pod Security admission for finer-grained and flexible policy control](https://github.com/kyverno/KDP/blob/main/proposals/extend_pod_security_admission.md).

Related PR: https://github.com/kyverno/kyverno/pull/4364

### Slack discussion

_No response_

### LINKED ISSUE #620: [Enhancement] Add flag `protectManagedResources`
### Description

PR https://github.com/kyverno/kyverno/pull/4414 adds a new flag `protectManagedResources`

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by chipzoller**
## Related issue #
Closes #604 
Closes #620 

## Proposed Changes
* Adds docs on the new podSecurity sub-rule type in 1.8
* Adds deprecation note for `splitPolicyReports` flag
* Adds new container flags
* Remove old annotation notes

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/Writing policies/validate.md [hunk 1]
```diff
@@ -801,3 +801,308 @@ spec:
...
+      capabilities:
+        drop:
+        - ALL
+```
+
+Regardless of where the disallowed `type: Unconfined` field is specified, Kyverno allows the Pod.
+
+Multiple control names may be excluded by listing them individually keeping in mind the previously-described rules. Refer to the [Pod Security Standards documentation](https://kubernetes.io/docs/concepts/security/pod-security-standards/) for a listing of all present controls, restricted fields, and allowed values.
+
+### PSA Interoperability
```
Comment (realshuting) [inline content/en/docs/Writing policies/validate.md:1086]: Good ones to add!

**Review by realshuting — APPROVED** [review]
/lgtm

File: content/en/docs/Writing policies/validate.md [hunk 2]
```diff
@@ -801,3 +801,308 @@ spec:
...
 
 The manifest validation feature shares many of the same abilities as the [verify images](/docs/writing-policies/verify-images/) rule type. For a more thorough explanation of the available fields, use the `kubectl explain clusterpolicy.spec.rules.validate.manifests` command.
+
+## Pod Security
+
+Starting in Kyverno 1.8, a new sub-rule type called `podSecurity` is available. This sub-rule type dramatically simplifies the process of writing and applying [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) profiles and controls. By integrating the same libraries as used in Kubernetes' [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/), enabled by default in 1.23 and stable in 1.25, Kyverno is able to apply all or some of the controls and profiles in as simple as a single rule while providing capabilities not possible in Pod Security Admission. Standard `match` and `exclude` processing is available just like with other rules. This sub-rule type is enabled when a `validate` rule is written with a `podSecurity` object, detailed below.
```
Comment (JimBugwadia) [inline content/en/docs/Writing policies/validate.md:807]: Nit: "rule subtype" sounds better to me than "sub-rule type"

Comment (JimBugwadia) [inline content/en/docs/Writing policies/validate.md:807]: ...profiles in a single rule

File: content/en/docs/Writing policies/validate.md [hunk 3]
```diff
@@ -801,3 +801,308 @@ spec:
...
+Regardless of where the disallowed `type: Unconfined` field is specified, Kyverno allows the Pod.
+
+Multiple control names may be excluded by listing them individually keeping in mind the previously-described rules. Refer to the [Pod Security Standards documentation](https://kubernetes.io/docs/concepts/security/pod-security-standards/) for a listing of all present controls, restricted fields, and allowed values.
+
+### PSA Interoperability
+
+Kyverno's podSecurity validate sub-rule type and Kubernetes' Pod Security Admission (PSA) are compatible and may be used together in a single cluster with an understanding of where each begins and ends. These are a few of the most common strategies when employing both technologies.
+
+{{% alert title="Note" color="info" %}}
+Pods which are blocked by PSA in enforce mode do not result in an AdmissionReview request being sent to admission controllers. Therefore, if a Pod is blocked by PSA, Kyverno never gets told about it.
```
Comment (JimBugwadia) [inline content/en/docs/Writing policies/validate.md:1091]: Kyverno cannot apply policies to it.

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
Looks good! Added a few minor comments.

**Comment by chipzoller** [comment]
Addressed comments.

**Review by JimBugwadia — APPROVED** [review]

**Pull request merged**